d3a_interface/constants_limits.py

```diff
@@ -67,7 +68,7 @@ class StorageSettings:
         # possible range of state of charge
         INITIAL_CHARGE_RANGE = RangeLimit(10, 100)
 
-        BREAK_EVEN_RANGE = RangeLimit(0, 10000)
+        BREAK_EVEN_RANGE = RateRange(0, 10000)
```

Comment:
min/max are ok here, therefore you can keep the RangeLimit namedtuple

Reply:
`BREAK_EVEN_RANGE` isnt used anymore too, if you agree, I will remove it too